src/main/kotlin/org/dashj/platform/dpp/util/Entropy.kt

```diff
@@ -31,12 +31,17 @@ object Entropy {
 
     @JvmStatic
     fun generate(): ByteArray {
-        return mockEntroy ?: secureRandom.generateSeed(SEED_SIZE)
+        if (mockEntropy == null) {
+            val bytes32 = ByteArray(32)
+            secureRandom.nextBytes(bytes32)
+            return bytes32
+        }
+        return mockEntropy!!
```

Comment:
Previously we were using `generateSeed` instead of `nextBytes` to get a random value.